PKGBUILD

```diff
@@ -0,0 +1,20 @@
+# Maintainer: Shreyansh Khajanchi <shreyansh_k@live.com>
+
+pkgname=gnirehtet
+pkgver=2.1
+pkgrel=1
+pkgdesc="Gnirehtet provides reverse tethering for Android"
+arch=('x86_64')
+url="https://github.com/Genymobile/gnirehtet"
+license=('Apache-2.0')
+depends=('android-tools')
+source=("https://github.com/Genymobile/gnirehtet/releases/download/v$pkgver/gnirehtet-rust-linux64-v$pkgver.zip")
+md5sums=('8f7cc0d33248cd6d71591d90e51ebbc0')
```

Comment:
MD5 is broken, use SHA-256 instead:

    sha256sums=('0f2a694611270eaf8a18af9ebf713932e05e4be75d0a38774154804da4d60d4d')